f/EandEcheck/f_EandECheckLoop.sqf

```diff
@@ -34,23 +31,12 @@ waitUntil {sleep 0.1;scriptDone f_script_setLocalVars};
 // POPULATE UNITS ARRAY
 // Conduct several checks against the first variable to see if we're dealing with a specific unit, a group or an array of several groups or units.
 
-// If a side was passed, just use the global variable generated by F3
+// If a side was passed, take all Units from that side
 if (_objects isEqualType sideUnknown) then {
-	switch (_objects) do {
-			case west: 			{_units = f_var_men_BLU};
-			case blufor: 		{_units = f_var_men_BLU};
-			case east: 			{_units = f_var_men_OPF};
-			case opfor: 		{_units = f_var_men_OPF};
-			case resistance: 	{_units = f_var_men_RES};
-			case independent: 	{_units = f_var_men_RES};
-			case civilian: 		{_units = f_var_men_CIV};
-	};
-
-	// If only players should be checked for, we process the units array to remove any unwanted units
 	if (_playersonly) then {
```

Comment:
Might aid readability to assign a variable, say `_possibleUnits`, conditionally, and then do `_possibleUnits select { (side _x) == _objects}` instead of repeating the `select`